index.html

```diff
@@ -3472,11 +3517,20 @@ <h3 id="applying">
     </section>
     <section>
```

Comment:
Add:
`data-dfn-for= "ExternalApplicationResource"` and `data-link-for="ExternalApplicationResource"`

Reply:
Please make sure all sections that define IDL things have both.